hr_employee_firstname/models/hr_employee.py

```diff
@@ -1,83 +1,108 @@
 # -*- coding: utf-8 -*-
 # ©  2010 - 2014 Savoir-faire Linux (<http://www.savoirfairelinux.com>)
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+# Copyright 2016-2019 Onestein (<https://www.onestein.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
-from odoo import models, fields, api
+import logging
+
+from odoo import api, fields, models
```

Comment:
```suggestion
from odoo import api, fields, models, _
```

To avoid `from odoo.tools.translate import _`